app/views/devise/mailer/confirmation_instructions.html.slim

```diff
@@ -1,4 +1,4 @@
-p.lead= t('mailer.confirmation_instructions.header', intro: @first_sentence)
+p.lead == t('mailer.confirmation_instructions.header', intro: @first_sentence)
```

Comment:
How would you feel about treating the i18n key as HTML-safe? If we rename the key to `header_html` then we don't need to remember to `==` here and can use `=`?